### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by daynewlee**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: compliance/collection/main.go [hunk 1]
```diff
@@ -277,13 +277,9 @@ func main() {
...
 		var scanner nodeinventorizer.NodeInventorizer
-		if features.UseFakeNodeInventory.Enabled() {
-			log.Infof("Using FakeNodeInventorizer")
-			scanner = &nodeinventorizer.FakeNodeInventorizer{}
-		} else {
-			log.Infof("Using NodeInventoryCollector")
-			scanner = &nodeinventorizer.NodeInventoryCollector{}
-		}
+
+		log.Infof("Using NodeInventoryCollector")
```
Comment (RTann) [inline compliance/collection/main.go:281]: probably no need for this log anymore

File: compliance/collection/main.go [hunk 2]
```diff
@@ -277,13 +277,9 @@ func main() {
 		go manageSendingToSensor(ctx, cli, sensorC)
 
 		var scanner nodeinventorizer.NodeInventorizer
```
Comment (RTann) [inline compliance/collection/main.go:279]: can remove this now

File: compliance/collection/main.go [hunk 3]
```diff
@@ -277,13 +277,9 @@ func main() {
...
-			log.Infof("Using FakeNodeInventorizer")
-			scanner = &nodeinventorizer.FakeNodeInventorizer{}
-		} else {
-			log.Infof("Using NodeInventoryCollector")
-			scanner = &nodeinventorizer.NodeInventoryCollector{}
-		}
+
+		log.Infof("Using NodeInventoryCollector")
+		scanner = &nodeinventorizer.NodeInventoryCollector{}
 		nodeInventoriesC := manageNodeScanLoop(ctx, env.NodeRescanInterval.DurationSetting(), scanner)
```
Comment (RTann) [inline compliance/collection/main.go:283]: probably no need to pass in a scanner anymore, unless we want to keep the fake one around for testing purposes?

**Review by RTann — COMMENTED** [review]
Seems fine, but I am wondering if we want to keep the fake stuff around for potential testing. I'll defer that to others more involved with the day-to-day tasks for RHCOS scanning.

Also, be sure to rename the PR :)

**Review by vikin91 — CHANGES_REQUESTED** [review]
I am only guessing that this PR is for https://issues.redhat.com/browse/ROX-13935. This ticket disables the feature flag and it is meant to be worked on when the feature is ready to release. We are not ready yet, to we cannot merge this.

Moreover, I think this PR should be a draft as the title and descriptions are empty. We do not want to ask for reviewers attention when they do not know what is to be reviewed.

**Pull request closed without merging**